src/Components/online_info.jsx

```diff
@@ -48,9 +47,11 @@ class OnlineInformation extends Component {
   }
 
   render() {
+    /*
```

Comment:
why did you comment out this timeout error, i think this is necessary when fetch fails.

Reply:
This is a good video to understand the concept: https://youtu.be/Gjnup-PuquQ

Reply:
Just throwing an error is not a good way to do error treatment, this just breaks the application, maybe you could present an alert to the user temporarily, before adding real error treatment.

Reply:
Yeah, but I am not getting the concept of how to remove the error that is enclosed by the ErrorBoundary component in react to catch that error

Reply:
I also want to continue the the application after the error is thrown by just rendering a button for trying to refetch the data by clicking on it. Here we can reset the error by clicking the button and then try to fetch same data again. But I am not getting how to implement that using errorboundaries in react. Could you please help me.

Reply:
You probably should use React State Hook for this: https://reactjs.org/docs/hooks-state.html
I don't have much experience in frontend development, so I don't know exactly how you should implement it, but using the application state is probably the best approach.